src/gradualizer_prelude.erl

```diff
@@ -2,8 +2,70 @@
 
 %% This module contains specs to replace incorrect or inexact specs in OTP.
 
+-type ext_binary() :: binary().
+
+-spec erlang:apply(function(), [any()]) -> any().
+-spec erlang:apply(atom(), function(), [any()]) -> any().
+-spec erlang:binary_to_term(ext_binary()) -> any().
+-spec erlang:binary_to_term(ext_binary(), [safe | unsafe]) -> any() | {any(), pos_integer()}.
```

Comment:
The options are `safe | used`, not ~~`unsafe`~~, according to http://erlang.org/doc/man/erlang.html#binary_to_term-2

Reply:
Thanks. Fixed.